lsp-mode.el

```diff
@@ -4056,9 +4058,13 @@ If ACTION is not set it will be selected from `lsp-code-actions'."
     (lsp--apply-formatting edits)))
 
 (defun lsp-organize-imports ()
-  "Perform the source.organizeImports code action."
+  "Perform the source.organizeImports code action, if available."
   (interactive)
-  (lsp-execute-code-action-by-kind "source.organizeImports"))
+  (condition-case nil
+      (lsp-execute-code-action-by-kind "source.organizeImports")
+    (lsp-no-code-actions
+     (when (called-interactively-p 'any)
+       (message "source.organizeImports action not available")))))
```

Comment:
Nit: can you change `message` to `lsp--info`?

Reply:
Done.